sensor/extrinsic_mapping_based_calibrator/src/calibration_mapper.cpp

```diff
@@ -265,7 +265,7 @@ void CalibrationMapper::mappingThreadWorker()
     frame->pointcloud_subsampled_ = PointcloudType::Ptr(new PointcloudType());
     PointcloudType::Ptr aligned_cloud_ptr(new PointcloudType());
     PointcloudType::Ptr cropped_cloud_ptr =
-      cropPointCloud<PointcloudType>(frame->pointcloud_raw_, parameters_->mapping_max_range_);
+      cropPointCloud<PointcloudType>(frame->pointcloud_raw_, 0.0, parameters_->mapping_max_range_);
```

Comment:
Although we don't use the troublesome LiDARs for mapping, can you see any benefit in also using a `mapping_min_range` parameter here just in case?

Reply:
@drwnz 
Addressed in [cf84be1](https://github.com/tier4/CalibrationTools/pull/107/commits/cf84be1fe36ffff2551b08d77e5ef32e73d5f006)